index.html

```diff
@@ -21,6 +27,23 @@
         </nav>
       </header>
 
+      <div class="mobile-menu glass">
+        <img class="menu-x" src="assets/mobile-menu-Icon-x.png" alt="menu x" />
+        <ul class="mobile-menu-ul">
+          <li>
+            <a class="mobile-menu-option" href="#works_container">Portfolio</a>
+          </li>
+          <li>
+            <a class="mobile-menu-option" href="#about_main_container">About</a>
+          </li>
+          <li>
+            <a class="mobile-menu-option" href="#contact_form_container"
+              >Contact</a
+            >
+          </li>
+        </ul>
+      </div>
+
```

Comment:
- [ ] Kindly make sure that when your application loads the mobile menu is not visible on the desktop version, please see the recording below to see how your app behaves 👍🏿 


https://github.com/Mehr-o-maah/Portfolio_2.github.io/assets/69638013/8aa851ad-a0f1-4857-9418-dc75004d8ceb